pyqir-generator/src/python.rs

```diff
@@ -5,102 +5,130 @@ use pyo3::{
     basic::CompareOp,
     exceptions::{PyOSError, PyTypeError},
     prelude::*,
+    types::PySequence,
     PyObjectProtocol,
 };
 use qirlib::generation::{
     emit,
     interop::{
-        ClassicalRegister, Controlled, If, Instruction, Measured, QuantumRegister, Rotated,
-        SemanticModel, Single,
+        Call, ClassicalRegister, Controlled, FunctionType, If, Instruction, Measured,
+        QuantumRegister, ReturnType, Rotated, SemanticModel, Single, Value, ValueType,
     },
 };
 use std::{
-    collections::hash_map::DefaultHasher,
+    collections::{hash_map::DefaultHasher, HashMap},
```

Comment:
It looks like there are already a lot of warnings from the spellchecker: "Files checked: 91, Issues found: 309 in 55 files." They are also too easy to ignore since the PR status shows it as succeeding. I think we need to deal with this for the whole repo separately from this PR if we want to remove the existing false positives and make sure new ones aren't added. I think @idavis added the spellchecker originally?

Reply:
@samarsha @cgranade  #49 took care of many of them and #20 is a higher tracker for dealing with more of the false positives. The spell checker was created from the [org .github template](https://github.com/qir-alliance/.github)

Reply:
I definitely would not suggest fixing unrelated issues in this PR, but made the comment specifically because the term "hasher" was causing a warning in this line.

Reply:
I understand, but I would prefer to wait for the spellchecking situation to be improved overall. I'm not sure that these identifiers should be spellchecked at all, since they come from third-party libraries. The current PR requirements for the repo don't seem to require any action right now (the PR status succeeds).